PurchasesCoreSwift/Public/PurchaserInfo.swift

```diff
@@ -207,8 +200,25 @@ import Foundation
         )
     }
 
+    private let allPurchases: [String: [String: Any]]
+
```

Comment:
I typically group the same protection level + var/lets together without new lines. When I go from let to var, I put a new line between them. Same with computed properties, newline before/after.